config/prod/ecmonsen-emorypipeline-lambda.yaml

```diff
@@ -0,0 +1,22 @@
+template_path: "remote/ecmonsen-emorypipeline-lambda.yaml"
+stack_name: "ecmonsen-emorypipeline-lambda"
+dependencies:
+  - prod/ecmonsen-emorypipeline-ci.yaml
+parameters:
+  SourceAccountId: "186706593821"
+  SynapseStorageLocationId: "0"
+  SynapseNotifyUsers: "3399376"
+  Department: SysBio
+  Project: amp-ad
+  OwnerEmail: eva.monsen@sagebase.org
+hooks:
+  before_create:
+    # Note: After Sep 30, 2020 only "virtual-hosted" bucket URLs (beginning with https://<bucketname>.s3.amazonaws.com)
+    # are supported. See https://aws.amazon.com/blogs/aws/amazon-s3-path-deprecation-plan-the-rest-of-the-story/
+    # Note: ecmonsen_emorypipeline_ci_code_bucket is the bucket created in the ecmonsen-emorypipeline-ci stack
+    # and its current value is in ../config.yaml
+
+    # The following command gets the lambda artifacts bucket name from the dependency stack (echoing it for logging)
+    - !cmd curl -sw "%{url_effective}\n" https://ecmonsen-emorypipeline-ci-lambdaartifactsbucket-nu5xkcp8vax4.s3.amazonaws.com/lambda-uploader/develop/ecmonsen-emorypipeline-lambda.yaml --create-dirs -o templates/remote/ecmonsen-emorypipeline-lambda.yaml
```

Comment:
why do you need `-sw "%{url_effective}\n"` ? and should the source path be `develop`?  I thought you were using master?

Reply:
Just changed to master. Was using -s for silent curl, and -w to print the url that was downloaded. -w was more useful when the url was dependent on variables. Not as useful now that it's hard-coded, but good validation nonetheless.

Reply:
it seems like `-w` just spit back the source URL? it doesn't tell whether the download was successful nor does it tell you whether any data was downloaded at all...

```
> curl -sw "%{url_effective}\n" https://ecmonsen-emorypipeline-ci-lambdaartifactsbucket-nu5xkcp8vax4.s3.amazonaws.com/lambda-uploader/master/foo.yaml -o bar.yaml
https://ecmonsen-emorypipeline-ci-lambdaartifactsbucket-nu5xkcp8vax4.s3.amazonaws.com/lambda-uploader/master/foo.yaml
```

Reply:
Removed it.